addons/mail/static/tests/helpers/test_utils.js

```diff
@@ -30,31 +31,38 @@ const { prepareTarget, nextTick } = testUtils;
  * @returns {Promise<Discuss>}
  */
 async function createDiscuss(params) {
-    var Parent = Widget.extend({
+    const target = prepareTarget(params.debug);
+    const Parent = Widget.extend({
         do_push_state: function () {},
     });
-    var parent = new Parent();
+    const parent = new Parent();
     params.archs = params.archs || {
         'mail.message,false,search': '<search/>',
     };
-    testUtils.mock.addMockEnvironment(parent, params);
+    await testUtils.mock.addMockEnvironment(parent, params);
```

Comment:
I don't get why `addMockEnvironment` has been turned asynchronous.
It seems to be motivated by services `start()` methods, but all of them are synchronous so...